dist/profile/templates/jenkinscontroller/casc/clouds.yaml.erb

```diff
@@ -456,7 +448,7 @@ jenkins:
                 '@ | Set-Content C:\ProgramData\Docker\config\daemon.json
 
                 # Restart docker engine
-                Restart-Service docker
```

Comment:
Damn, I messed up this line (unrelated to the PR: I did a PEBKAC in the GitHub UI). Gotta restore it.